packages/components/src/components/iot-table/iot-table.tsx

```diff
@@ -11,7 +11,9 @@ import {
   ProviderWithViewport,
 } from '@iot-app-kit/core';
 import { v4 as uuidv4 } from 'uuid';
-import { createTableItems, Item, TableProps } from '@iot-app-kit/table';
+import { createTableItems, Item, TableProps, DefaultTableMessages, TableMessages } from '@iot-app-kit/table';
+import merge from 'lodash/merge';
+import { RecursivePartial } from '../../testing/types';
```

Comment:
move `RecursivePartial` to `utils/types` or some other location that makes more sense. shouldn't use testing stuff in implementing components